src/pages/loginView.js

```diff
@@ -1,41 +1,68 @@
 import React, {useState} from 'react';
+import { Link, useHistory } from 'react-router-dom';
+
 import Button from '../components/global/Button';
 import TextInput from '../components/global/TextInput';
 import LoginImage from '../assets/images/LoginImage';
-import {Link} from 'react-router-dom';
-
-import '../stylesheets/loginView.scss';
 
 export default function LoginView() {
+  const history = useHistory();
+
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
-  const login = function () {
-    console.log(email, password);
+  const [ emailInfoText, setEmailInfoText ] = useState("");
+  const [ passwordInfoText, setPasswordInfoText ] = useState("");
+  
+  const login = () => {
+    // Process inputs
+    const processedEmail = email.trim();
+    const processedPassword = password.trim();
+    
+    // 
+    let validLogin = true;
+    if (!processedEmail) {
+      setEmailInfoText("Please enter your email address");
+      validLogin = false;
+    } 
+    if (!processedPassword) {
+      setPasswordInfoText("Please enter your password");
+      validLogin = false;
+    }
+
+    if (validLogin) {
+      history.push("/home");
+    }
+    
   };
 
   return (
     <>
       <div className='login'>
-        <h1 className='welcome'>Welcome</h1>
-        <div className='email_container'>
-          <TextInput label='Email:' onChangeHandler={setEmail} type='email' />
+        <h1 className='login__welcome'>Welcome</h1>
+        <div className='email-container'>
+          <TextInput 
+            label='Email:' 
+            placeholderValue={emailInfoText}
+            onChangeHandler={setEmail} 
+            type='email' />
         </div>
-        <div className='password_container'>
+        <div className='password-container'>
           <TextInput
             label='Password:'
+            placeholderValue={passwordInfoText}
             onChangeHandler={setPassword}
             type='password'
           />
         </div>
-        <div onClick={login}>
-          <Button icon={'rightArrow'} text={'Login'} />
+        <Button icon='rightArrow' text='Login' handleOnClick={login} />
+        <div className="login__bottom-text">
+          <span>Don&apos;t have an account?</span>
```

Comment:
Just curious why a normal ' doesn't work here?

Reply:
this was a linting suggestion, I think's for browser/device compatability